_posts/courses/intro-to-earth-data-science-textbook/07-efficient-clean-code-python/loops/2020-07-03-loops-python-05-loops-exercise.md

```diff
@@ -0,0 +1,314 @@
+---
+layout: single
```

Comment:
@nkorinek can you please have a look at your pr? it looks like the exercise name is not right or it did not build. it should be named:

_posts/courses/intro-to-earth-data-science-textbook/07-efficient-clean-code-python/loops/2020-07-03-loops-05-loops-exercise.md